cost/aws/old_snapshots/aws_delete_old_snapshots.pt

```diff
@@ -229,15 +306,43 @@ escalation "esc_delete_snapshot" do
       description "Expain why are you approving the action"
     end
   end
-  run "delete_snapshot",data
```

Comment:
Since we are waiting for the new selectable actions feature this shouldn't have the approval block and it shold have the label and description

Reply:
@cdwilhelm, The following changes have been taken care of in [POL-253](https://jira.flexera.com/browse/POL-253).